CONTRIBUTING.rst

```diff
@@ -142,6 +142,12 @@ It's easy to add files to version control but forget to include in the MANIFEST.
     $ check-manifest
 
 
+Updating the getting started notebook
+-------------------------------------
+
+The project contains a notebook to help orientate new users. You can modify this notebook in the demo project and update the template with:
```

Comment:
I'm a little confused by this - can we not just update the `getting_started.ipynb` notebook directly? Is there a demo project in this repo outside of that one?

Reply:
> I'm a little confused by this - can we not just update the getting_started.ipynb notebook directly?

You can update the template notebook, but that template notebook will not work as it contains placeholder values like `{{cookiecutter.project_name}}` that are not valid Python. It's better to work with a notebook in a created project where you can validate it runs then convert it back to a template.

The template: https://github.com/BookletAI/whisk/blob/7741efdcecdecab12ca7c57a7dc0c61893d50879/whisk/template/%7B%7B%20cookiecutter.repo_name%20%7D%7D/notebooks/getting_started.ipynb